### PR TITLE
[ch914] bigquery: adding support for executing queries

### PR DESCRIPTION
**Pull request by damjanovl**

### CODE REVIEW

File: database/bigquery/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pandas==1.0.4
```
Comment (ericelsken) [inline database/bigquery/requirements.txt:1]: Is pandas a "transient" dependency. Required by bigquery? It isn't being referenced in the code.

File: database/bigquery/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+google-cloud-bigquery-1.25.0
```
Comment (ericelsken) [inline database/bigquery/requirements.txt:2]: Is this the correct syntax? Does Google put the version in the name of package?

Comment (damjanovl) [reply database/bigquery/requirements.txt]: Fixed

File: database/bigquery/requirements.txt (hunk 1 above)
Comment (damjanovl) [reply database/bigquery/requirements.txt]: Used this during testing and forgot to remove it from the requiements.txt. Removed it!

**Pull request merged**